packages/core/src/components/image-overlay/image-overlay.tsx

```diff
@@ -11,6 +18,12 @@ export class ImageOverlay implements ComponentInterface {
   @Element()
   host!: HTMLDsoImageOverlayElement;
 
+  /**
+   * The wijzigactie as in STOP.
```

Comment:
Laat het STOP aspect maar achterwege, Image Overlay heeft daar niets mee te maken. Dat wij het mappen op STOP functionaliteit (in een STOP-aware component) is iets anders